android/src/main/JNI/Android.mk

```diff
@@ -37,8 +37,10 @@ ifeq ($(NATIVE_DEBUG), true)
     cmd-strip :=
 endif
 
-LOCAL_MODULE := turbomodulejsijni
-include $(BUILD_SHARED_LIBRARY)
+ifeq ($(shell test $(REACT_NATIVE_TARGET_VERSION) -lt 64; echo $$?),0)
```

Comment:
Do not use shell here. The variable is already available.

Reply:
I'm not aware of any boolean env variable defining if we should make old or new style of build. Are you suggesting creating one?